.travis.yml

```diff
@@ -18,4 +18,5 @@ before_install:
 install:
   - bin/buildout -N -t 3
 script:
+  - flake8 --config travis_ci_flake8.cfg src/
```

Comment:
should this become a separate job? currently it takes < 1min to run, so personally I think this is fine.